### ISSUE TITLE
HtmlCrawlerSubscriber ignores <base> Tag

### ISSUE THREAD

**Issue by Mynyx** [terminal42/escargot#11]
The ``HtmlCrawlerSubscriber`` does not consider the ``<base>`` tag for relative links.

**Comment by Toflar** [comment]
Do you have an example? I can then use this for my tests and check what happens right away :)

**Comment by fritzmg** [comment]
Aren't all of Contao's links relative to the base tag for example?

**Comment by Toflar** [comment]
Apparently it's all good because it works just fine currently. So I do need an example of what exactly doesn't work.

**Comment by Mynyx** [comment]
> Do you have an example? I can then use this for my tests and check what happens right away :)

I have observed this in relation to another issue, but here is a simplified example:

````
https://developer.mozilla.org/de/docs/Web/HTML/Element

HTTP/2.0 200 OK
content-type: text/html; charset=UTF-8

<html>
    <head>
        <base href="https://developer.mozilla.org/de/docs/Web/HTML/Element/">
    </head>
    <body>
        <a href="base">Link</a>
    </body>
</html>
````

Should trigger a successful request to https://developer.mozilla.org/de/docs/Web/HTML/Element/base

**Comment by Toflar** [comment]
Fixed in fd49c10dfd170519e15c012f45dcceba22f3b9ca.
